<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>410</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>410</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="openCloseButton">
   <property name="geometry">
    <rect>
     <x>324</x>
     <y>170</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>OPEN</string>
   </property>
  </widget>
  <widget class="QLabel" name="parity">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>Parity:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="dataBitsBox">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>80</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="databits">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>DataBits:</string>
   </property>
  </widget>
  <widget class="QLabel" name="baudrate">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>BaudRate:</string>
   </property>
  </widget>
  <widget class="QLabel" name="port">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>Port:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="stopBitsBox">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>110</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="stop">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>StopBits:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="baudRateBox">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>50</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="portBox">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>20</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="parityBox">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>140</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </widget>
  <widget class="HLed" name="led" native="true">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>170</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>20</width>
     <height>400</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTextBrowser" name="recvEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>140</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="fileOpenButton">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>25</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="dataBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>225</width>
     <height>280</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="startStopButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pauseResumeButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>28</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>Upload data from file</string>
   </property>
  </widget>
  <zorder>dataBrowser</zorder>
  <zorder>openCloseButton</zorder>
  <zorder>parity</zorder>
  <zorder>dataBitsBox</zorder>
  <zorder>databits</zorder>
  <zorder>baudrate</zorder>
  <zorder>port</zorder>
  <zorder>stopBitsBox</zorder>
  <zorder>stop</zorder>
  <zorder>baudRateBox</zorder>
  <zorder>portBox</zorder>
  <zorder>parityBox</zorder>
  <zorder>led</zorder>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
  <zorder>recvEdit</zorder>
  <zorder>fileOpenButton</zorder>
  <zorder>startStopButton</zorder>
  <zorder>pauseResumeButton</zorder>
  <zorder>label</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>HLed</class>
   <extends>QWidget</extends>
   <header>hled.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
